Framework/CurveFitting/test/FuncMinimizers/FABADAMinimizerTest.h

```diff
@@ -190,23 +190,23 @@ class FABADAMinimizerTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(PDF->getNumberHistograms(), nParams + 1);
     TS_ASSERT_EQUALS(PDF->x(0).size(), 21);
     TS_ASSERT_EQUALS(PDF->y(0).size(), 20);
-    TS_ASSERT_DELTA(PDF->y(0)[7], 0.41, 0.01);
-    TS_ASSERT_DELTA(PDF->y(1)[8], 5.24, 0.01);
-    TS_ASSERT_DELTA(PDF->y(2)[0], 0.44, 0.01);
+    TS_ASSERT_DELTA(PDF->y(0)[7], 0.41, 0.3);
+    TS_ASSERT_DELTA(PDF->y(1)[8], 5.24, 1.0);
+    TS_ASSERT_DELTA(PDF->y(2)[0], 0.44, 0.3);
```

Comment:
In fact asymptotically it is "deterministic", but the test cannot be performed with large numbers of iterations since it has to run quickly and large numbers imply longer run time.

The test for the minimum works, since it is a single point and it corresponds to the minimum chi^2 visited, which is easily recorded and the probability to get around that is fairly high and not so many iterations are needed. For the rest of the pdf to be equal it is not so easy, a lot of variance should be expected since not so many iterations are considered and it is not a single point value but an integral instead, thus a much longer time is needed for it to average correctly, specially for low probability points